new/LIT14518QeneOzyan.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT14518QeneOzyan" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ዖዝያን፡ ወድቀ፡ እመንበረ፡ ተድላ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʿOzyān wadqa ʾǝm-manbara tadlā... (Qǝne possibly of kwǝllǝkǝmu-type)</title>
+                <title xml:lang="en" corresp="#t1">Uzziah fell from the thrown of glory…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne with an allusion to the fall of Uzziah as described in <ref type="work" corresp="LIT3500Chroni">2 Chr. 26</ref> and 
+                    other biblical books. The type and author are not given in the manuscript. The verses end in ላ፡ and are marked with ።, ending up in 
+                    a series of possibly nine verses, so that the poem can be attributed the kwǝllǝkǝmu type of qǝne poetry according to the 
```

Comment:
Could you please make the last sentence a bit more concise? "including two verse markers in `<ref target="l2"/>` and `<ref target="l2"/>`, which are not visible in the available pictures because of the tight binding of the manuscript)." - this string is shown with mark-up and doe not need further explanation. I would use less "possibly" etc. as it is now nine verses, not ten or eight.  It is understandable that anyone who wishes to count it differently can do so.